apps/language_server/lib/language_server/ex_unit_test_tracer.ex

```diff
@@ -76,11 +76,19 @@ defmodule ElixirLS.LanguageServer.ExUnitTestTracer do
         test_info
         |> Enum.group_by(fn %ExUnit.Test{tags: tags} -> {tags.describe, tags.describe_line} end)
         |> Enum.map(fn {{describe, describe_line}, tests} ->
+          grouped_tests =
+            tests |> Enum.group_by(fn %ExUnit.Test{tags: tags} -> tags.test_type end)
```

Comment:
this doesn't need a pipeline
```suggestion
          grouped_tests =  Enum.group_by(tests, fn %ExUnit.Test{tags: tags} -> tags.test_type end)
```